platform/common/spinning_zsync

```diff
@@ -30,7 +30,7 @@ fi
     #SPINNER="▌ ▛ ▀ ▜ ▐ ▟ ▄ ▙"
     while :; do
         for spin in ${SPINNER}; do
-            usleep 500000
+            usleep 500000 2>/dev/null || sleep 0.5
```

Comment:
I'm actually somewhat surprised that the BusyBox for Kobo supports fractional sleep.

Reply:
Yeah, for reference, to recap what was said on gitter:

* usleep is an "all or nothing" applet
* fractional sleep support is optional

FW 2.x (and possibly 3.x, too) on Kindle builds busybox with usleep, but without fractional sleep support ;).